content/messaging/agent-chat-api/v3.2/index.mdx

```diff
@@ -1170,59 +1170,56 @@ curl -X POST \
 
 ```json
 {
-  "chat": {
 	"id": "PJ0MRSHTDG",
 	"users": [{
-	  "id": "b7eff798-f8df-4364-8059-649c35c9ed0c",
-	  "type": "customer",
-	  "present": true,
-	  "created_at": "2019-12-17T08:53:20.693553+01:00",
-	  "statistics": {
-		"chats_count": 1,
-		// ...
-	  },
-	  "agent_last_event_created_at": "2019-12-17T09:04:05.239000+01:00"
+		"id": "b7eff798-f8df-4364-8059-649c35c9ed0c",
+		"type": "customer",
+		"present": true,
+		"created_at": "2019-12-17T08:53:20.693553+01:00",
+		"statistics": {
+			"chats_count": 1
+		},
+		"agent_last_event_created_at": "2019-12-17T09:04:05.239000+01:00"
 	}, {
-	  "id": "smith@example.com",
-	  "name": "Agent Smith",
-	  "email": "smith@example.com",
-	  "type": "agent",
-	  "present": true,
-	  "avatar": "https://example.com/avatar.jpg"
-	}],
-	"threads": [{
-	  "id": "K600PKZON8",
-	  "active": true,
-	  "user_ids": [
-		"b7eff798-f8df-4364-8059-649c35c9ed0c",
-		"smith@example.com"
-	  ],
-	  "events": [{
-		"id": "Q20N9CKRX2_1",
-		"created_at": "2019-12-17T07:57:41.512000Z",
-		"recipients": "all",
-		"type": "message",
-		"text": "Hello",
-		"author_id": "smith@example.com"
-	  }, ],
-	  "properties": {
-		// "Property" object
-	  },
-	  "access": {
-		"group_ids": [0]
-	  },
-	  "previous_thread_id": "K600PKZOM8",
-	  "next_thread_id": "K600PKZOO8",
-	  "created_at": "2020-05-07T07:11:28.288340Z"
+		"id": "smith@example.com",
+		"name": "Agent Smith",
+		"email": "smith@example.com",
+		"type": "agent",
+		"present": true,
+		"avatar": "https://example.com/avatar.jpg"
 	}],
+	"thread": {
+		"id": "K600PKZON8",
+		"active": true,
+		"user_ids": [
+			"b7eff798-f8df-4364-8059-649c35c9ed0c",
+			"smith@example.com"
+		],
+		"events": [{
+			"id": "Q20N9CKRX2_1",
+			"created_at": "2019-12-17T07:57:41.512000Z",
+			"recipients": "all",
+			"type": "message",
+			"text": "Hello",
+			"author_id": "smith@example.com"
+		}],
+		"properties": {
```

Comment:
I'll be soon moving all responses to individual JSON files. If possible, I'd like us to gradually stop using those "JSON comments". Could we replace it with:
```
     "properties": {
          "0805e283233042b37f460ed8fbf22160": {
              "string_property": "string_value"
              }
          }
```
or something similar